english/33hours-add-on/slow-recall.md

```diff
@@ -0,0 +1,2 @@
+The cold chill of fear gives way to your inside voice..."dammit I had it figured out!".  Sitting on the edge of the bed you hold your aching head and speak out loud, "but what is it?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.